app/scripts/modules/amazon/src/loadBalancer/loadBalancer.transformer.ts

```diff
@@ -134,21 +134,21 @@ export class AwsLoadBalancerTransformer {
       isInternal: loadBalancer.isInternal,
       region: loadBalancer.region,
       cloudProvider: loadBalancer.cloudProvider,
-      credentials: loadBalancer.account,
-      listeners: [],
+      credentials: loadBalancer.credentials,
+      listeners: loadBalancer.listeners,
       loadBalancerType: 'classic',
       name: loadBalancer.name,
       regionZones: loadBalancer.availabilityZones,
-      securityGroups: [],
-      vpcId: undefined,
+      securityGroups: loadBalancer.securityGroups,
+      vpcId: loadBalancer.vpcId,
       healthCheck: undefined,
-      healthTimeout: undefined,
-      healthInterval: undefined,
-      healthyThreshold: undefined,
-      unhealthyThreshold: undefined,
-      healthCheckProtocol: undefined,
-      healthCheckPort: undefined,
-      healthCheckPath: undefined,
+      healthTimeout: loadBalancer.healthTimeout,
+      healthInterval: loadBalancer.healthInterval,
+      healthyThreshold: loadBalancer.healthyThreshold,
+      unhealthyThreshold: loadBalancer.unhealthyThreshold,
+      healthCheckProtocol: loadBalancer.healthCheckProtocol,
+      healthCheckPort: loadBalancer.healthCheckPort,
+      healthCheckPath: loadBalancer.healthCheckPath,
```

Comment:
would like @jrsquared to take a look at this part - this seems reasonable to me (and if they're undefined, they'll get set further down in the method) but would like a little more validation from someone who's looked at this code more recently and is more familiar with it.

Reply:
Looks reasonable to me. I want to pull this down and test it first though. I should have time in the next couple of days.

Reply:
These need to be set during the edit, or their values aren't carried over from what was previously set (they are set by default when you create the load balancer, so setting them back to undefined here would delete whatever config had been initially saved).